packages/shared/src/components/post/common/PostBackButton.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactElement } from 'react';
+import { useRouter } from 'next/router';
+import { ButtonVariant } from '../../buttons/common';
+import ArrowIcon from '../../icons/Arrow';
+import { Button } from '../../buttons/ButtonV2';
+import { FeedLayout } from '../../../lib/featureValues';
+import { useFeature } from '../../GrowthBookProvider';
+import { feature } from '../../../lib/featureManagement';
+
+interface PostBackButtonProps {
+  link?: string;
+}
+
+export function PostBackButton({ link }: PostBackButtonProps): ReactElement {
+  const router = useRouter();
+  const layout = useFeature(feature.feedLayout);
```

Comment:
Same as above here, or does it not have the feed context?

Reply:
Yes, there is no context of feed here.